src/OperationalInsights/OperationalInsights/Client/OperationalInsightsClient.Tables.cs

```diff
@@ -45,33 +46,132 @@ public virtual List<PSTable> FilterPSTables(string resourceGroupName, string wor
             return tables;
         }
 
-        public virtual PSTable UpdatePSTable(UpdatePSTableParameters parameters)
+        /// <summary>
```

Comment:
any reason these methods are virtual in this class?

Reply:
removed